bindings/nodejs/lib/types/wallet/transaction-options.ts

```diff
@@ -13,20 +19,24 @@ export interface TransactionOptions {
     remainderValueStrategy?: RemainderValueStrategy;
     /** An optional tagged data payload. */
     taggedDataPayload?: TaggedDataPayload;
-    /**
-     * Custom inputs that should be used for the transaction.
-     * If custom inputs are provided, only those are used.
-     * If also other additional inputs should be used, `mandatoryInputs` should be used instead.
-     */
-    customInputs?: OutputId[];
+    /** Transaction context inputs to include. */
+    contextInputs?: ContextInput[];
     /** Inputs that must be used for the transaction. */
-    mandatoryInputs?: OutputId[];
+    requiredInputs?: OutputId[];
     /** Specifies what needs to be burned during input selection. */
     burn?: Burn;
     /** Optional note, that is only stored locally. */
     note?: string;
     /** Whether to allow sending a micro amount. */
     allowMicroAmount?: boolean;
+    /** Whether to allow the selection of additional inputs for this transaction. */
+    allowAdditionalInputSelection?: boolean;
+    /** Transaction capabilities. */
+    capabilities?: HexEncodedString;
+    /** Mana allotments for the transaction. */
+    manaAllotments?: { [account_id: AccountId]: u64 };
```

Comment:
Would that `account_id` actually be named in the serialisation ?

Reply:
no, I don't think so